.github/workflows/codeformat.yaml

```diff
@@ -1,16 +1,17 @@
-name: pull_request
+name: Code format check
 on: pull_request
 jobs:
   codegen:
     name: Code format check
     runs-on: ubuntu-22.04
     steps:
       - name: Cancel Previous Actions
-        uses: styfle/cancel-workflow-action@0.11.0
+        uses: styfle/cancel-workflow-action@0.12.1
```

Comment:
I've upgraded each action as there doesn't appear to be any particular reason to use old actions